### PR TITLE
[Leetcode]: Valid Parentheses #366

### PR DESCRIPTION
**Pull request by nisargaa20**
## Related Issue

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

## Description

[Please include a brief description of the changes or features added]

added leetcode solution 20.Valid Parentheses
Fixed: #366

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/0000-0099/0020-valid.parentheses.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+#### Example 3
+
+- **Input:** `s = "(]"`
+- **Output:** false
+
+### Constraints
+
+- $1 <= s.length <= 104$
+- $s consists of parentheses only '()[]{}'.$
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0020-valid.parentheses.md:47]: replace `- $s consists of parentheses only '()[]{}'.$` to `- s consists of parentheses only '()[]{}'.`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code only change one line

**Comment by nisargaa20** [comment]
@Ajay-Dhangar changed and javascript solution too

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**